cvcl/models.py

```diff
@@ -284,3 +284,58 @@ class Meta:
 
     def __str__(self):
         return str(self.id)
+
+
+class InstallPackage(models.Model):
+    name = models.CharField(max_length=30)
+    package_update = models.BooleanField(default=False)
```

Comment:
@DSpeichert  Wouldn't it be better to create another model with the fields for package_update, package_upgrade, and package_reboot_if_required and create a relationship with this new model and the model we have for installing Packages.